crossword-xml-uploader/src/main/scala/com/gu/crossword/crosswords/XmlProcessor.scala

```diff
@@ -2,10 +2,21 @@ package com.gu.crossword.crosswords
 
 import java.util.Locale
 import scala.xml._
-import org.joda.time.format.{ DateTimeFormat }
+import org.joda.time.format.{ DateTimeFormat, ISODateTimeFormat }
 import org.joda.time.{ DateTimeZone, LocalDateTime }
 
-trait XmlProcessor {
+trait DateLogic {
+
+  def transformDate(dateString: String): String = {
+    val inputFormat = DateTimeFormat.forPattern("dd.MM.yyyy HH:mm")
+    val outputFormat = ISODateTimeFormat.dateTime()
+    val isoFormattedOutput = LocalDateTime.parse(dateString, inputFormat).toDateTime(DateTimeZone.forID("Europe/London")).toString(outputFormat)
+    isoFormattedOutput.replaceAllLiterally("Z", "+00:00")
```

Comment:
You mean why do I replace the Z? Just to be consistent with the existing behaviour in the non-summertime case. It's probably not necessary.